src/datepicker/bs-daterangepicker.component.ts

```diff
@@ -110,6 +110,10 @@ export class BsDaterangepickerDirective
    * Disable specific dates
    */
   @Input() datesDisabled: Date[];
+  /**
+   * Enable specific dates
+   */
+  @Input() datesEnabled: Date[];
```

Comment:
Why this property is mandatory for this directive? Is it correct?

Reply:
replied above